platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterUtils.java

```diff
@@ -66,6 +70,20 @@ public static Hash hash(@NonNull final Roster roster) {
         return PBJ_RECORD_HASHER.hash(roster, Roster.PROTOBUF);
     }
 
+    /**
+     * Build a map from a long nodeId to a RosterEntry for a given Roster.
+     *
+     * @param roster a roster
+     * @return Map&lt;Long, RosterEntry&gt;
```

Comment:
I hate writing escaped angle brackets.  The `{@code Map<Long, RosterEntry>}` might work here.   See the second and third solutions to this stack overflow: https://stackoverflow.com/questions/2898897/how-can-i-use-and-in-javadoc-without-formatting

Reply:
I can change it to `{@code}` in the next PR.